### PR TITLE
Add E2E tests for `/decision/`

### PR DESCRIPTION
**Pull request by Timothy-Gonzalez**
<Just testing, this will probably fail>

### CODE REVIEW

File: tests/e2e/decision_test/decision_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	if err != nil {
+		fmt.Printf("ERROR: %v\n", err)
+		os.Exit(1)
+	}
+
+	admin_client = common.GetSlingClient("Admin")
+
+	client = common.GetLocalMongoSession()
+
+	decision_db_name, err := cfg.Get("DECISION_DB_NAME")
```
Comment (Nydauron) [inline tests/e2e/decision_test/decision_test.go:32]:
```suggestion
	decision_db_name, err = cfg.Get("DECISION_DB_NAME")
```

File: tests/e2e/decision_test/get_decision_test.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+				Status:    "PENDING",
+				Wave:      1,
+				Reviewer:  "bob",
+				Timestamp: 123,
+				ExpiresAt: 1234,
+			},
+		},
+	}
+
+	client.Database(decision_db_name).Collection("profile").InsertOne(context.Background(), &decision_info)
```
Comment (Nydauron) [inline tests/e2e/decision_test/get_decision_test.go:34]:
```suggestion
	client.Database(decision_db_name).Collection("decision").InsertOne(context.Background(), &decision_info)
```

**Review by Nydauron — COMMENTED** [review]
2 changes that will fix the issue. Also in the future, make sure to make a separate branch off of `staging` and push your changes there.

**Comment by Timothy-Gonzalez** [comment]
Superseded by #499

**Pull request closed without merging**